src/Screen/Fields/Cropper.php

```diff
@@ -30,6 +30,12 @@
  * @method Cropper popover(string $value = null)
  * @method Cropper title(string $value = null)
  * @method Cropper maxFileSize($value = true)
+ * @method Cropper minWidth($value = null)
+ * @method Cropper minHeight($value = null)
+ * @method Cropper maxWidth($value = null)
+ * @method Cropper maxHeight($value = null)
+ * @method Cropper min($value = null)
+ * @method Cropper max($value = null)
```

Comment:
These are real methods, there is no need to specify additional meta attributes

Reply:
cleared!